packages/account/src/providers/transaction-request/input.ts

```diff
@@ -97,7 +97,7 @@ export const inputify = (value: TransactionRequestInput): Input => {
       return {
         type: InputType.Coin,
         txID: hexlify(arrayify(value.id).slice(0, 32)),
-        outputIndex: arrayify(value.id)[32],
+        outputIndex: toNumber(arrayify(value.id).slice(32, 33)),
```

Comment:
Could you also convert these to named constants as well?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2034/commits/7fc0e27d3f5bc103001248f7f2dbc8211c16f00c